layers/theming/README.org

```diff
@@ -0,0 +1,90 @@
+* Theming contribution layer for Spacemacs
+
+** Table of Contents                                                  :TOC@4:
+ - [[#theming-contribution-layer-for-spacemacs][Theming contribution layer for Spacemacs]]
+   - [[#description][Description]]
+   - [[#install][Install]]
+   - [[#usage][Usage]]
+   - [[#attributes][Attributes]]
+   - [[#faces][Faces]]
+   - [[#headers][Headers]]
+
+** Description
+This layer allows for a simple way of modifying themes.
+
+** Install
+To use this contribution add it to your =~/.spacemacs=
+
+#+begin_src emacs-lisp
+  (setq-default dotspacemacs-configuration-layers '(theming))
+#+end_src
+
+** Usage
+To use this layer, set the value of =theming-modifications=. It should be a list
```

Comment:
Almost...

```elisp
(setq theming-modifications
      '((subatomic (default :background "#272B38"))))
```

And it needs to go in `dotspacemacs/user-init`, since it needs to be set before the theme is loaded.